STEP_Project/OUListContent.js

```diff
@@ -1,10 +1,43 @@
+let OBJ;
+if (chrome.storage != undefined)
+{
+    OBJ = chrome.storage.sync;
+}
+
 /**
  * Function that will later have Add/Remove functionality
  * @param {JS event} event - expect click event
  */
 function addRemoveButtonClick (event) 
 {
-    if (event.target.getAttribute("class")!=='bClass') return;
+    let classname = event.target.getAttribute('class')
+    if (classname !="pClass" && classname !="mClass")
+    {
+        return;
+    }
+
+    let row = event.target.parentElement;
+    var dataid = row.getAttribute("data-row-id");
+
+    if (classname=="pClass")
+    { 
+        event.target.innerHTML="-";
+        event.target.setAttribute('class','mClass'); 
+        let dataname = "name";
```

Comment:
This was done for testing purposes. So if the entire DOM tree is not present exactly like the admin website (as in our mock table object in the testing file), we set the default to "name", otherwise it's set to the real name.

Reply:
I think the correct way in that case would be to not save at all, rationale being the same I mentioned in the comment above.

Reply:
How about using an empty string as the default dataname, again for testing purposes? For admin using the actual extension, it will never be the default, it will always be the actual name, so we would not have to worry about the name not meaning anything to the admin, or any repeats.

Reply:
I think even for tests it would be better to assert that nothing gets stored it relevant information is not there in the DOM.

Reply:
- In my test script, I'm using a mock table. The mock table does not have children[0].children[0].children[0].children[1].innerHTML. I'm not asserting that the _dataname_ has any value in my test script, I only assert that it's not undefined, just so that I know it exists in the storage.
- I changed the default to an empty string so that repeats is not a problem.